src/controller.js

```diff
@@ -1,36 +1,30 @@
 const db = require("./db");
 const queries = require("./queries");
 const { dateToDB, shorten } = require("./util.js");
+const linkRepository = require("link_repository")
 
-const postEntry = (req, res) => {
+const postEntry = async (req, res) => {
   const url = req.body.url;
-  const now = dateToDB();
+  const now = dateToDB(); // you can take utc time from server no need to ping DB
```

Comment:
dateToDB() is a utility function that converts the JS time format into a SQL-appropriate timezone format.

Reply:
Ah I see. It's just a conversion of a date object into an ISO date format string. That is not 'postgresql' specific, is a date format standard, the function should reflect the action better, for example 'dateToISO' in this case.